core-library/src/main/java/org/silverpeas/core/util/PdfUtil.java

```diff
@@ -98,11 +171,11 @@ public static void watermark(InputStream pdfSource, File watermarkImage,
   private static void addImageOnEachPage(File pdfSource, File image, File pdfDestination,
       final boolean isBackground) {
     if (pdfSource == null || !pdfSource.isFile()) {
-      throw new RuntimeException("The pdf source file doesn't exist");
+      throw new SilverpeasRuntimeException(PDF_FILE_ERROR_MSG);
     } else if (!FileUtil.isPdf(pdfSource.getPath())) {
-      throw new RuntimeException("The source is not a pdf file");
+      throw new SilverpeasRuntimeException("The source is not a pdf file");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'NOT_PDF_FILE_ERROR_MSG' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)